sig-cloud-provider/2020-annual-report.md

```diff
@@ -0,0 +1,72 @@
+# 2020 - SIG Cloud Provider - Community Meeting Annual Reports
+---
+## Operational
+**How are you doing with operational tasks in sig-governance.md?**
+
+**Is your README accurate? have a CONTRIBUTING.md file?**
+
+Our README is accurate but we do not currently have a contributing.md file.
+
+**All subprojects correctly mapped and listed in [sigs.yaml](https://github.com/kubernetes/community/blob/master/sigs.yaml)?**
+
+I believe so.
+
+**What’s your meeting culture? Large/small, active/quiet, learnings? Meeting notes up to date? Are you keeping recordings up to date/trends in community members watching recordings?**
+
+Our meetings tends to be small and quiet. We try to keep the meeting notes upto date during the meeting. 
+
+**How does the group get updates, reports, or feedback from subprojects? Are there any springing up or being retired? Are OWNERS files up to date in these areas?**
+_Same question as above but for working groups._
+
+We go ove the sub project updates during our primary biweekly meeting. 
+I believe the OWNERS files are all upto date.
+
+**When was your last monthly community-wide update? (provide link to deck and/or recording)**
+
+---
+## Membership
+**Are all listed SIG leaders (chairs, tech leads, and subproject owners) active?**
+
+I believe so.
+
+**How do you measure membership? By mailing list members, OWNERs, or something else?**
+
+?
+
+**How does the group measure reviewer and approver bandwidth? Do you need help in any area now? What are you doing about it?**
+
+?
+
+**Is there a healthy onboarding and growth path for contributors in your SIG? What are some activities that the group does to encourage this? What programs are you participating in to grow contributors throughout the contributor ladder?**
+
+Most of our SIG members are contributors from at least one cloud provider. We try to reach our to the various cloud providers during KubeCon to make sure they are aware of us and that their voices are being heard. New members come in slowly enough that we can generally provide direct mentorship and arrange this through our slack channel or meetings.
+
+**What programs do you participate in for new contributors?**
+
+?
+
+**Does the group have contributors from multiple companies/affiliations? Can end users/companies contribute in some way that they currently are not?**
+
+We try hard to make sure we have presentation from each of the interested cloud providers.
+
+---
+## Current initiatives and project health
+_Please include links to KEPs and other supporting information that will be beneficial to multiple types of community members._
+
+**What are initiatives that should be highlighted, lauded, shout out, that your group is proud of? Currently underway? What are some of the longer tail projects that your group is working on?**
+
+?
+
+**Year to date KEP work review: What’s now stable? Beta? Alpha? Road to alpha?**
+
+TBD
```

Comment:
https://github.com/kubernetes/enhancements/issues/2436 to alpha
(add cred. provider too?)